bendowen/BEN_DOWEN.md

```diff
@@ -0,0 +1,74 @@
+# Tester of The Day
+
+## Origin story
+
+The story starts, as with all good things, with a joke. Dan Ashby had started a rant on Twitter about Paid-for awards, around the end of July 2020.
+Lot's of people joined in and thread and the jokes rumbled on for a few days. Many of us making more outranges sounding awards.
+
+You can checkout the thread for yourself: <https://twitter.com/DanAshby04/status/1288763581333413888>
+
+Inspired by the humour of this all, I thought, what if? What if I made a daily award, a free award?
```

Comment:
How about? `Inspired by the humour of this all, I thought, "what if?". What if I made a daily award? What if it was a free award?
`